conf.d/hydro.fish

```diff
@@ -43,6 +43,7 @@ function _hydro_prompt --on-event fish_prompt
     for code in $last_status
         if test $code -ne 0
             set _hydro_prompt "$_hydro_color_error"[(string join "\x1b[2mǀ\x1b[22m" $last_status)]
+            test "$hydro_show_prompt_symbol_on_error" = true && set --append _hydro_prompt "$_hydro_color_prompt_error$hydro_symbol_prompt_error"
```

Comment:
I'm not sure about this. I'm new to fish and there doesn't seem to be a proper way to check for a truthy value (see [here](https://github.com/fish-shell/fish-shell/issues/7120)).

But this requires the variable set to literally `true` (so `1`, for example, won't work), so it might not be ideal.
Let me know if you want this changed to something else.